frameworks/kafka/docs/kerberos.md

```diff
@@ -1,5 +1,9 @@
 # Setting up Kafka with Kerberos
 
+Still TODO:
+1. The KDC deployment still requires the IP to be specified
```

Comment:
Once #1794 is merged, the KDC address can be resolved via `kdc.marathon.autoip.dcos.thisdcos.directory` and the port will be configurable.